cvat/apps/engine/views.py

```diff
@@ -189,6 +189,12 @@
 _DATA_UPDATED_DATE_HEADER_NAME = 'X-Updated-Date'
 _RETRY_AFTER_TIMEOUT = 10
 
+ICON_FILE = "assets/logo.svg"
+
+def get_logo() -> str:
```

Comment:
Please check that we don't read it on every about request.

Reply:
Changed implementation to use static logo file as Roman suggested. Now it should be cached correctly